test/RibbonThetaVault.ts

```diff
@@ -1134,1263 +1092,787 @@ function behavesLikeRibbonOptionsVault(params: {
-        const depositAmount = BigNumber.from("100000000000");
-
-        await depositIntoVault(params.collateralAsset, vault, depositAmount);
-
-        const startETHBalance = await assetContract.balanceOf(user);
-
-        await depositIntoVault(
-          params.collateralAsset,
-          vault.connect(counterpartySigner),
-          BigNumber.from("1000000000000")
-        );
-
-        // As the pool expands, using 1 pool share will redeem more amount of collateral
-        const res = await vault.withdraw(depositAmount, {
-          gasPrice,
-        });
-        await res.wait();
-
-        assert.equal(
-          (await assetContract.balanceOf(user)).sub(startETHBalance).toString(),
-          BigNumber.from("99500000000").toString()
-        );
-      });
-
-      it("should withdraw more collateral when the balance increases", async function () {
-        const depositAmount = BigNumber.from("100000000000");
-
-        await depositIntoVault(params.collateralAsset, vault, depositAmount);
-
-        const startAssetBalance = await assetContract.balanceOf(user);
-
-        await depositIntoVault(
-          params.collateralAsset,
-          vault.connect(counterpartySigner),
-          BigNumber.from("1000000000000")
-        );
-
-        await depositIntoVault(
-          params.collateralAsset,
-          vault.connect(counterpartySigner),
-          BigNumber.from("1000000000000")
-        );
-
-        // As the pool expands, using 1 pool share will redeem more amount of collateral
-        const res = await vault.withdraw(depositAmount, {
-          gasPrice,
-        });
-        await res.wait();
-
-        assert.equal(
-          (await assetContract.balanceOf(user))
-            .sub(startAssetBalance)
-            .toString(),
-          BigNumber.from("99500000000")
-        );
-      });
-
-      it("should revert if not enough shares", async function () {
-        const depositAmount = BigNumber.from("100000000000");
-
-        await depositIntoVault(params.collateralAsset, vault, depositAmount);
-
-        await depositIntoVault(
-          params.collateralAsset,
-          vault.connect(counterpartySigner),
-          BigNumber.from("1000000000000")
-        );
-
-        await expect(
-          vault.withdraw(depositAmount.add(BigNumber.from("10000000")))
-        ).to.be.revertedWith("ERC20: burn amount exceeds balance");
-      });
-
-      it("should be able to withdraw everything from the vault, leaving behind minimum", async function () {
-        const depositAmount = BigNumber.from("100000000000");
-
-        await depositIntoVault(params.collateralAsset, vault, depositAmount);
-
-        // simulate setting a bad otoken
-        await vault.connect(managerSigner).commitAndClose();
-
-        // users should have time to withdraw
-        await vault.withdraw(depositAmount.sub(await vault.minimumSupply()));
-      });
```

Comment:
should we just do it.skip() and then change tests?

Reply:
Will do

Reply:
Actually, do you mind ignoring this for now, will be doing so in the next PR which includes the rollToNextOption tests